### ISSUE TITLE
[Accessibility] Back to top button is not accessible

### ISSUE THREAD

**Issue by phucduong86** [CodingTrain/website#2444]
The back-to-top button is not accessible to keyboard users:

- The button is not focusable. Recommend using native button or adding ```tabindex="0"```
- State indicator (CSS) should be added to the button on hover/focus
- ```title``` attribute behaves differently on different screen readers so when possible ```aria-label``` is preferred
- This is probably not too important (and doesn't violate any regulations) since the website's footer is relatively small in size, however the 'back to top' button is usually placed right above the footer.

![image](https://user-images.githubusercontent.com/21692768/88063543-22c27d80-cb38-11ea-94dd-c6198a98b934.png)

**Comment by phucduong86** [comment]
I just took a quick look on the website, styling and focus works great.
Please also add keydown handler on the button (currently there's only click handler) so the keyboard user can activate the button.
